pkg/api/mlflow/dao/repositories/run.go

```diff
@@ -237,3 +259,37 @@ func (r RunRepository) SetRunTagsBatch(ctx context.Context, run *models.Run, bat
 	}
 	return nil
 }
+
+// getMinRowNum will find the lowest row_num for the slice of runs
+// or 0 for an empty slice
+func getMinRowNum(runs []models.Run) int64 {
```

Comment:
one more thing: you could get a slice of runs that have the zero value for their `RowNum` because they did not exist in the first place. This could all be avoided by checking that no bad IDs were passed by the user before we delete them.